cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        while True:
+            try:
+                connection.ensure_connection()
+            except OperationalError:
+                self.stdout.write(
+                    "Failed to find database"
+                )
+                time.sleep(5)
+            else:
+                break
```

Comment:
The `break` statement is used to exit the loop when the database connection is ensured. However, it would be more readable and pythonic to use a `return` statement instead of `break` in this case. This will immediately exit the function, making it clear that no further processing will occur.